tutorials/math/interpolation.rst

```diff
@@ -138,4 +142,48 @@ Here is how it looks:
 
 .. image:: img/interpolation_follow.gif
 
-This is useful for smoothing camera movement, allies following you (ensuring they stay within a certain range), and many other common game patterns.
+This is useful for smoothing camera movement, for allies following the player
+(ensuring they stay within a certain range), and for many other common game patterns.
+
+.. note::
+ Despite using ``delta``, the formula used above is framerate-dependent, because
+ the ``weight`` parameter of ``lerp()`` represents a *percentage* of the remaining
+ difference in values, not an *absolute amount to change*. In ``_physics_process()``,
+ this is usually fine because physics is expected to maintain a constant framerate,
+ and therefore ``delta`` is expected to remain constant. For example, with an FPS
+ of ``60`` and a ``FOLLOW_SPEED`` of ``4.0``, ``delta`` will be ``1/60`` or ``0.0166``,
+ and ``FOLLOW_SPEED * delta`` will be a constant value of ``0.0666``.
```

Comment:
This last sentence feels perhaps unecessary, but I wanted to illustrate that even with `delta` involved, the value `FOLLOW_SPEED * delta` tends to be constant.

Reply:
I'd lean towards removing the last sentence here. Examples help when they provide a simple illustration of a complex concept, here, `In _physics_process() this is usually fine because physics is expected to maintain a constant framerate` is easier for me to parse and conceptualize than the example with all those numbers.

Reply:
Removed.